tests/VocabularyConfigTest.php

```diff
@@ -515,6 +515,16 @@ public function testGetOrderedPlugins() {
     $this->assertEquals(["plugin2", "Bravo", "imaginaryPlugin", "plugin1", "alpha", "charlie", "plugin3"],  $plugins);
   }
 
+  /**
+   * @covers VocabularyConfig::getPluginArray
+   */
+  public function testGetUnorderedVocabularyPlugins() {
+    $vocab = $this->model->getVocabulary('paramPluginOrderTest');
+    $plugins = $vocab->getConfig()->getPluginArray();
+    $arrayElements = ["plugin2", "Bravo", "imaginaryPlugin", "plugin1", "alpha", "charlie", "plugin3"];
+    $this->assertEquals(sort($arrayElements), sort($plugins));
```

Comment:
Doesn't this double sort (sorting both sides) kind of defy the purpose of the issue this PR is supposed to fix?